backend/company_api.py

```
@@ -0,0 +1,776 @@
+    limit: int = Query(100, ge=1, le=1000),
+    offset: int = Query(0, ge=0),
+    user: dict = Depends(_get_current_user_thunk)
+) -> SpecialistListResponse:
+    """
+    List all specialists for a company.
+    
+    Returns specialists provisioned for the specified company.
+    """
+    company = await get_company_access(company_id, user)
+    
+    specialist_service = get_specialist_service()
```

Ruff (0.15.14)

[error] Undefined name `get_specialist_service`

(F821)